# Generated by Django 3.2.12 on 2022-03-15 05:52

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('menu', '0006_auto_20220315_0548'),
    ]

    operations = [
        migrations.AddField(
            model_name='menuitem',
            name='size',
            field=models.CharField(choices=[('L', 'L'), ('XL', 'XL')], default='L', max_length=2),
            preserve_default=False,
        ),
        migrations.AlterField(
            model_name='category',
            name='name',
            field=models.CharField(error_messages={'unique': 'That category already exists.'}, max_length=255, unique=True, validators=[django.core.validators.MinLengthValidator(3)]),
        ),
        migrations.AlterField(
            model_name='ingredient',
            name='name',
            field=models.CharField(error_messages={'unique': 'That ingredient already exists.'}, max_length=255, unique=True, validators=[django.core.validators.MinLengthValidator(3)]),
        ),
    ]
